script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -702,7 +702,7 @@ It happen again? You collapse from anemia this morning or something?"
 -- Page 122, Offset 1473.
 -- 心配してくれるのはありがたいんだけどね、そう四六時中貧血を起こしてたら体がもたないよ」
 It's nice of you to worry about me,
- but if I were anemic 24/7 my body would have withered away by now."
+ but if I were anemic 24/7, my body would have withered away by now."
```

Comment:
This 2 comma construction basically always strikes me as 'textbook' and overly slow for short spoken sentences like this

Reply:
yeah that's why i thought it was correct :necoarc:
ill refrain from adding those in voiced lines if it doesn't work